TaskLayer/SearchTask/SearchTask.cs

```diff
@@ -78,6 +78,12 @@ public static MassDiffAcceptor GetMassDiffAcceptor(Tolerance precursorMassTolera
 
         protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask> dbFilenameList, List<string> currentRawFileList, string taskId, FileSpecificParameters[] fileSettingsList)
         {
+            //disable quantification if a .mgf is being used
+            if (SearchParameters.DoQuantification && currentRawFileList.Any(x => Path.GetExtension(x).Equals(".mgf")))
```

Comment:
this is ok for now but at some point I'll correct FlashLFQ to bail on quantifying files without MS1 spectra. just making a mental note here.

you probably want to use OrdinalIgnoreCase when checking for ".mgf"